Maple2.File.Parser/Xml/Table/PremiumClubItem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+using M2dXmlGenerator;
+
+namespace Maple2.File.Parser.Xml.Table;
+
+// ./data/xml/table/vipbenefititemtable.xml
+[XmlRoot("ms2")]
+public class PremiumClubItemRoot {
+    [XmlElement] public List<PremiumClubItem> benefit;
+}
+
+public partial class PremiumClubItem : IFeatureLocale {
+    [XmlAttribute] public int id;
+    [XmlAttribute] public int itemID;
+    [XmlAttribute] public int itemCount;
+    [XmlAttribute] public int itemRank;
+    [XmlAttribute] public int itemPeriod;
```

Comment:
`itemPeriod` is `long`
![image](https://user-images.githubusercontent.com/7710340/200245921-9267e55c-c229-4052-8e24-6bb57d5b8431.png)